engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -93,11 +91,13 @@ private void updateFullScale() {
      */
     public void update() {
         updateFullScale();
-        if (READ_ONLY_GBUFFER.getFbo().dimensions().areDifferentFrom(fullScale)) {
+        if (get(new ResourceUrn("engine:sceneOpaque")).dimensions().areDifferentFrom(fullScale)) {
```

Comment:
Let's first get the FBO in a line before the IF, the we use it. And by now you know what to do with the ResourceUrn.